backend/core/src/auth/dto/user-update.dto.ts

```diff
@@ -70,12 +70,6 @@ export class UserUpdateDto extends OmitType(UserDto, [
   @IsNotEmpty({ groups: [ValidationsGroupsEnum.default] })
   currentPassword?: string
 
-  @Expose()
-  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
-  @ValidateNested({ groups: [ValidationsGroupsEnum.default], each: true })
-  @Type(() => IdDto)
-  jurisdictions: IdDto[]
-
```

Comment:
As a super admin and jurisdictional admin, I should be able to update the jurisdictions a user has access to. For the jurisdictional admins, that means we'll have to make sure that they have access to the jurisdictions they're updating.